frontend/javascripts/types/api_flow_types.ts

```diff
@@ -360,6 +360,8 @@ type APIAnnotationBase = APIAnnotationCompact & {
   readonly restrictions: APIRestrictions;
   readonly viewConfiguration?: AnnotationViewConfiguration | null | undefined;
   readonly settings: APISettings;
+  readonly owner?: APIUserBase;
```

Comment:
Is the owner field really optional?

Reply:
Yes, I think, the owner is not set if the current user does not have the permissions to view the user details (e.g., when it's public). However, this might change with #6230. @fm3 correct me if I'm wrong :)

Reply:
I think that’s right, yes. We should double-check again as part of #6230 that every case gets a valid response, and see how much info about the user is required where.